bin/metrics/cli.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+const {main} = require('./main');
+const meow = require('meow');
+
+const command = meow({
+  description: 'CLI tool for collecting styleguide metrics.',
+  help: `
+    Usage
+      $ yarn sg-metrics <...options> paths
```

Comment:
Following bash conventions it should be:
`yarn sg-metrics [options] <paths>`

[] - optional param
<> - mandatory param

Reply:
@clxandstuff

Reply:
👍